tests/jobs/ahm_analysis/test_ahm_analysis.py

```diff
@@ -0,0 +1,418 @@
+import os
+
+import pytest
+import pandas as pd
+import numpy as np
+
+import semeio.workflows.ahm_analysis.ahmanalysis as ahmanalysis
+from semeio._exceptions.exceptions import ValidationError
+
+from unittest.mock import MagicMock
+from itertools import product
+from pathlib import Path
+
+from scipy import stats
+from ecl.grid import EclGridGenerator
+
+
+@pytest.mark.usefixtures("setup_tmpdir")
```

Comment:
Don't think we need a tmpdir here, we are only reading from a file using pandas.

Reply:
I guess we need the tempdir since the functions search files in a dir...

Reply:
It does, but the path provided to `make_update_log_df` is simply a path in source dir (as we are only reading, and not writing, that will suffice): `root/semeio/tests/legacy_test_data/update_log/allobs`

Reply:
ok, I update this

Reply:
done